projects/objects/street_furniture/protos/Fence.proto

```diff
@@ -237,21 +236,19 @@ PROTO Fence [
               let coordIndexBuffer = '';
               for (let i = 0; i <= nbPoles - 1; ++i) {
                 // Top part
-                for (let j = 0; j <= circleSubdivision; ++j)
+                for (let j = circleSubdivision; j >= 0; --j)
                   coordIndexBuffer += i * skip + circleSubdivision - j + ' ';
                 coordIndexBuffer += ' -1\n';
 
                 // Bottom part
-                for (let j = 0; j <= circleSubdivision; ++j)
+                for (let j = circleSubdivision; j >= 0; --j)
                   coordIndexBuffer += i * skip + j + circleSubdivision + 1 + ' ';
                 coordIndexBuffer += ' -1\n';
 
                 // Sides
                 for (let j = 0; j <= circleSubdivision - 1; ++j) {
-                  coordIndexBuffer += i * skip + j + ' ';
-                  coordIndexBuffer += i * skip + j + 1 + ' ';
-                  coordIndexBuffer += i * skip + j + circleSubdivision + 2  + ' ';
-                  coordIndexBuffer += i * skip + j + circleSubdivision + 1 + ' -1\n';
+                  const start = i * skip + j;
```

Comment:
The Fence PROTO is especially slow in JavaScript compared to Lua, and is also used to define things like `CrashBarrier` so although template literals are more readable, it's best to build the geometry in a buffer for performance reasons. This PROTO is one among those that added a huge overhead to world like `village_center`.

Reply:
Actually, it has a performance impact :)
https://jsbench.me/0skt9pgzus/1